runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/util/MonitoringUtil.java

```diff
@@ -180,14 +180,15 @@ public int compare(JobMessage o1, JobMessage o2) {
     return allMessages;
   }
 
-  public static String getJobMonitoringPageURL(String projectName, String jobId) {
+  public static String getJobMonitoringPageURL(String projectName, String regionId, String jobId) {
```

Comment:
Let's do what you propose: "overload the method and sub in "us-central1" by default which should be sufficient to avoid breaking end user code"